frida_tools/lsd.py

```diff
@@ -19,9 +19,21 @@ def _start(self) -> None:
                 self._update_status(f"Failed to enumerate devices: {e}")
                 self._exit(1)
                 return
+            device_os = {}
+            device_name = {}
+            for device in devices:
+                device_name[device.id] = device.name
+                try:
+                    params = device.query_system_parameters()
+                    device_name[device.id] = params.get("name", device.name)
+                    os = params["os"]
+                    device_os[device.id] = os["name"] + " " + os.get("version", "")
```

Comment:
Would be good to avoid emitting trailing whitespace in case `version` is missing.